src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1002,17 +1002,29 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
             matchOffs.push_back(matchOff);
             replaceMatchOns.push_back(replaceMatchOn);
             replaceMatchOffs.push_back(replaceMatchOff);
-            for (std::string newError : errorMessage)
-               errors.push_back(json::Value(newError));
+            {
```

Comment:
Why are these statements in a block?

Reply:
I have a habit (from style guidelines in a past life) of scoping variables only used for a line or two. In this case it really doesn't matter since the scope ends two lines later so I removed the block.